python/rpdk/java/codegen.py

```diff
@@ -10,12 +11,30 @@
 from rpdk.core.plugin_base import LanguagePlugin
 
 from .resolver import translate_type
-from .utils import safe_reserved, validate_namespace
+from .utils import safe_reserved, validate_codegen_model, validate_namespace
 
 LOG = logging.getLogger(__name__)
 
 OPERATIONS = ("Create", "Read", "Update", "Delete", "List")
 EXECUTABLE = "cfn"
+AWSCODEGEN = namedtuple("AWSCODEGEN", "default guided default_code guided_code")
+CODEGEN = AWSCODEGEN("default", "guided_aws", "1", "2")
```

Comment:
what is default here ?

Reply:
also what are these other ones? I only see anything about guided_aws in here

Reply:
this generates a simple class with four properties: 
```python
__dict__: 
    {
        default: '/default', 
        guided: '/guided', 
        default_code: '1', 
        guided_code: '2',
}
```
codes are to validate user inputs from the prompt and associate correct directory (`/default` or `/guided`) to generate code from the stubs


ln:21 is setting associated defaults. 
@surega21 `default` is a directory path to the standard stubs
@aygold92  this is used in order to specify paths and codes once and reuse across the class.

this function uses this class - `_prompt_for_codegen_model` (ln:90)
 |- `guided_aws` is the directory path for the new stubs, it is referenced via `CODEGEN.guided` ln:100
this `_is_aws_guided` checks if we are in guided mode (ln:108)